### ISSUE TITLE
Plotting in Jupyter returns AttributeError: 'NoneType' object has no attribute 'has_capabilities' 

### ISSUE THREAD

**Issue by rhoP** [pyvista/pyvista#6149]
### Describe the bug, what's wrong, and what you expected.

I'm using pyvista on Windows 11. My code runs fine on my work laptop running fedora, but a similar python environment in windows is unable to render plots due to the above error. 

The full error details are:

AttributeError                            Traceback (most recent call last)
Cell In[3], [line 8](vscode-notebook-cell:?execution_count=3&line=8)
      [6](vscode-notebook-cell:?execution_count=3&line=6) p = pv.Plotter()
      [7](vscode-notebook-cell:?execution_count=3&line=7) p.add_mesh(mesh)
----> [8](vscode-notebook-cell:?execution_count=3&line=8) p.show()

File c:\Users\Rohit\anaconda3\envs\pyg\lib\site-packages\pyvista\plotting\plotter.py:6734, in Plotter.show(self, title, window_size, interactive, auto_close, interactive_update, full_screen, screenshot, return_img, cpos, jupyter_backend, return_viewer, return_cpos, before_close_callback, **kwargs)
   [6731](file:///C:/Users/Rohit/anaconda3/envs/pyg/lib/site-packages/pyvista/plotting/plotter.py:6731)         jupyter_backend = self._theme.jupyter_backend
   [6733](file:///C:/Users/Rohit/anaconda3/envs/pyg/lib/site-packages/pyvista/plotting/plotter.py:6733)     if jupyter_backend.lower() != 'none':
-> [6734](file:///C:/Users/Rohit/anaconda3/envs/pyg/lib/site-packages/pyvista/plotting/plotter.py:6734)         jupyter_disp = handle_plotter(self, backend=jupyter_backend, **jupyter_kwargs)
   [6736](file:///C:/Users/Rohit/anaconda3/envs/pyg/lib/site-packages/pyvista/plotting/plotter.py:6736) self.render()
   [6738](file:///C:/Users/Rohit/anaconda3/envs/pyg/lib/site-packages/pyvista/plotting/plotter.py:6738) # initial double render needed for certain passes when offscreen

File c:\Users\Rohit\anaconda3\envs\pyg\lib\site-packages\pyvista\jupyter\notebook.py:31, in handle_plotter(plotter, backend, screenshot, **kwargs)
     [28](file:///C:/Users/Rohit/anaconda3/envs/pyg/lib/site-packages/pyvista/jupyter/notebook.py:28)     if backend in ['server', 'client', 'trame', 'html']:
     [29](file:///C:/Users/Rohit/anaconda3/envs/pyg/lib/site-packages/pyvista/jupyter/notebook.py:29)         from pyvista.trame.jupyter import show_trame
---> [31](file:///C:/Users/Rohit/anaconda3/envs/pyg/lib/site-packages/pyvista/jupyter/notebook.py:31)         return show_trame(plotter, mode=backend, **kwargs)
     [33](file:///C:/Users/Rohit/anaconda3/envs/pyg/lib/site-packages/pyvista/jupyter/notebook.py:33) except ImportError as e:
     [34](file:///C:/Users/Rohit/anaconda3/envs/pyg/lib/site-packages/pyvista/jupyter/notebook.py:34)     warnings.warn(
     [35](file:///C:/Users/Rohit/anaconda3/envs/pyg/lib/site-packages/pyvista/jupyter/notebook.py:35)         f'Failed to use notebook backend: \n\n{e}\n\nFalling back to a static output.'
     [36](file:///C:/Users/Rohit/anaconda3/envs/pyg/lib/site-packages/pyvista/jupyter/notebook.py:36)     )

File c:\Users\Rohit\anaconda3\envs\pyg\lib\site-packages\pyvista\trame\jupyter.py:385, in show_trame(plotter, mode, name, server_proxy_enabled, server_proxy_prefix, jupyter_extension_enabled, collapse_menu, add_menu, add_menu_items, default_server_rendering, handler, **kwargs)
    [382](file:///C:/Users/Rohit/anaconda3/envs/pyg/lib/site-packages/pyvista/trame/jupyter.py:382)     raise TrameServerDownError(name)
...
--> [376](file:///C:/Users/Rohit/anaconda3/envs/pyg/lib/site-packages/trame_vtk/widgets/vtk/common.py:376) self._helper.has_capabilities("web", "rendering")
    [378](file:///C:/Users/Rohit/anaconda3/envs/pyg/lib/site-packages/trame_vtk/widgets/vtk/common.py:378) __ns = kwargs.get("namespace", "view")
    [380](file:///C:/Users/Rohit/anaconda3/envs/pyg/lib/site-packages/trame_vtk/widgets/vtk/common.py:380) if __ns == "view":

AttributeError: 'NoneType' object has no attribute 'has_capabilities'

### Steps to reproduce the bug.

import pyvista as pv
from pyvista import examples

mesh = pv.Wavelet()

p = pv.Plotter()
p.add_mesh(mesh)
p.show()

### System Information

```shell
OS : Windows (10 10.0.22621 SP0 Multiprocessor Free)
              CPU(s) : 16
             Machine : AMD64
        Architecture : 64bit
                 RAM : 15.8 GiB
         Environment : Jupyter
          GPU Vendor : NVIDIA Corporation
        GPU Renderer : NVIDIA GeForce RTX 3050 Laptop GPU/PCIe/SSE2
         GPU Version : 4.5.0 NVIDIA 555.85
    MathText Support : False

  Python 3.10.14 | packaged by Anaconda, Inc. | (main, May  6 2024, 19:44:50)
  [MSC v.1916 64 bit (AMD64)]

             pyvista : 0.43.8
                 vtk : $PKG_VERSION
               numpy : 1.23.1
          matplotlib : 3.8.4
              scooby : 0.10.0
               pooch : 1.7.0
              pillow : 10.3.0
...

  Intel(R) oneAPI Math Kernel Library Version 2021.4-Product Build 20210904
  for Intel(R) 64 architecture applications
```

**Comment by jourdain** [comment]
Which version of `vtk` and `trame-vtk`?

**Comment by jourdain** [comment]
My feeling is that your VTK was not built with the WEB module on.

**Comment by rhoP** [comment]
The installed versions of some of these packages are:
vtk - 9.0.3
trame - 3.6.0
trame-client - 3.0.3
trame-server - 3.0.0
trame-vtk - 2.8.8

I was not aware about building vtk with the web module, as I am just installing the packages using conda. How would I go about doing this? Thanks :)

**Comment by jourdain** [comment]
You need `vtk>=9.2`. I think that is the point when the recipe on conda was fixed to enable that module. 
Ideally you should pick the latest VTK which is 9.3.

**Comment by rhoP** [comment]
I just installed vtk 9.3 but still get the same error, unfortunately. Also updated trame to 3.6.1. Is there something else I could try?

**Comment by jourdain** [comment]
The issue you described should have been fixed by using a modern vtk.

Can you provide the full steps so we could reproduce that issue?

**Comment by olly-writes-code** [comment]
I just came across this using the latest pyvista (0.46.4) and vtk (9.5.2)

For other people who come across this problem I fixed it by running

```
uv pip uninstall pyvista
uv pip uninstall vtk
uv sync
```

**Comment by olly-writes-code** [comment]
Noting that I'm running into this on every new machine I work with. I think this issue should be re-opened. I can provide a fuly reproducible example.

**Comment by jourdain** [comment]
Do you know which version of VTK is getting pulled?

**Comment by olly-writes-code** [comment]
Hi @jourdain! VTK 9.5.2.

**Comment by nixjdm** [comment]
> I just came across this using the latest pyvista (0.46.4) and vtk (9.5.2)
> 
> For other people who come across this problem I fixed it by running
> 
> ```
> uv pip uninstall pyvista
> uv pip uninstall vtk
> uv sync
> ```

I spent a while wondering what was going on until I found this issue. This fixed it for me, somehow. The output from this shows no change, so it's still a mystery to me. At least, this unblocked me, so thank you!

```
Uninstalled 1 package in 6ms
 - pyvista==0.44.1
Uninstalled 1 package in 8ms
 - vtk==9.4.1
Resolved 244 packages in 0.97ms
Installed 2 packages in 12ms
 + pyvista==0.44.1
 + vtk==9.4.1
```

**Comment by olly-writes-code** [comment]
@banesullivan flagging that other users are starting to see this so it might be worth re-opening the issue

**Comment by olly-writes-code** [comment]
Noting that I just installed a fresh venv with the newer trame versions and this issue didn't seem to come up. Will keep an eye on this and report back.

He's the change in trame versions

```
trame-common       1.0.1 -> 1.1.0      
trame-server       3.8.1 -> 3.9.0      
trame-vtk          2.10.0 -> 2.10.1     
```

@jourdain maybe theres a known fix in one of these ver bumps?

**Comment by jourdain** [comment]
I don't think so. That is a specific thing in VTK which could be missing on old VTK version or old conda bundle.
